packages/angular_devkit/build_angular/src/browser/schema.json

```diff
@@ -308,7 +308,8 @@
     "es5BrowserSupport": {
       "description": "Enables conditionally loaded ES2015 polyfills.",
       "type": "boolean",
-      "default": false
+      "default": false,
```

Comment:
I think we should remove the default which would allow false to actually mean disabled and undefined (not explicitly specified) means auto-detect.